boards/google/quincy/doc/index.rst

```diff
@@ -0,0 +1,47 @@
+.. _google_quincy_board:
+
+Google Quincy Development Board
+###################################
```

Comment:
please use board template -- ie. use `zephyr:board::` directive, and set a full_name in board.yml. Check twinke_v2, as an example of how to do it correctly

Reply:
Done, thanks, please re-open with concerns